ccmlib/node.py

```diff
@@ -232,7 +232,10 @@ def set_install_dir(self, install_dir=None, version=None, verbose=False):
             dir, v = setup(version, verbose=verbose)
             self.__install_dir = dir
         self.import_config_files()
-        self.import_bin_files()
+        if upgrade:
+            self.import_bin_files(exist_ok=True, replace=True)
```

Comment:
I don't see that `import_bin_files` has parameters

Reply:
import_bin_files() in ccmlib/scylla_node.py has those two parameters.